Uzira.rte/Actors/Brains/Uzira/Uzira.lua

```diff
@@ -0,0 +1,80 @@
+function Create(self)
+	self.minion = CreateAHuman("Skeleton", "Uzira.rte");
+	self.weapons = {"Uzira.rte/Blunderpop", "Uzira.rte/Blunderbuss", "Uzira.rte/Musket"};
```

Comment:
Any reason not to include other weapons in here?

Reply:
Wanted to only give the dinky guns to the auto-spawned skellies so that they don't use explosive weapons near the Brain.

Reply:
Should they perhaps start with an extra couple actors given how weak they are?

Reply:
The spawn rate is highest when no minions have yet been spawned, so it's essentially the same as having them spawn immediately 

Besides, I think Conquest spawns you with extra reinforcements whenever attacking with a high enough budget

Reply:
> Wanted to only give the dinky guns to the auto-spawned skellies so that they don't use explosive weapons near the Brain.

Should probably chuck in the crossbow then right? And I think maybe a small percentage of them having boomsticks would be cool, but maybe you're right and that's a poor decision haha